tf_encrypted/keras/models/sequential.py

```diff
@@ -101,3 +103,66 @@ def layers(self):
     if layers and isinstance(layers[0], InputLayer):
       return layers[1:]
     return layers[:]
+
+  def set_weights(self, keras_weights, sess):
+    """ Sets the weights of the model.
+    Arguments:
+      weights: A list of Numpy arrays with shapes and types
+          matching the output of model.get_weights()
+      sess: tfe session"""
+
+    # List model weights
+    model_weights = []
+    for l in self.layers:
+      model_weights += l.weights
+
+    # Define keras weights as private variables
+    keras_weights_p = [tf_encrypted.define_private_variable(w)
```

Comment:
we should not create new variables here, only assign to existing ones

Reply:
this should be implemented in the base Layer abstraction instead of at the model level (note that this class will inherit the method in that case, see the tf.keras source code for more)

Reply:
How would you assign a numpy array to an existing private variable?

Reply:
should we use a placeholder? something like `sess.run(v.assign_from_same(w), feed_dict=p.feed(np.array([5, 5, 5])))`

https://github.com/tf-encrypted/tf-encrypted/blob/c9200f3d814f54128c4bad24c615458dfc8f253d/examples/deprecated/int100.py

Reply:
in fact if use a placeholder we could something like:

```
new_weights_pl = tfe.define_private_placeholder(new_weights_shape)
fd = p.feed(new_weigths_array)
sess.run(tfe.assign(existing_variable_weights, new_weights_pl), feed_dict=fd)
```

Reply:
yeah it'd probably have to be a placeholder, so that sharing happens locally. we did some code for this around the queues, maybe that fits directly!

Reply:
ideally we'd move that placeholder into the tfe.Variable, so it can be re-used. problem with creating these objects (vars and placeholders) in the Keras code is that they add to the graph

Reply:
Makes sense @mortendahl .

Reply:
@jvmancuso good point, regarding `set_weights` in `base_layer`. Just added.